.github/actions/generic-ci/action.yml

```diff
@@ -180,6 +194,7 @@ runs:
         cmake --install . --component configuration
 
     - name: Build and configure libf3d examples
+      if: inputs.cpu == 'x86_64'
```

Comment:
This could be checked for arm I think

Reply:
You mean doing `inputs.cpu == 'arm64'` instead?  
I disagree. The runners are `x86_64`, so we need to check the architecture is the same as the runners.

Reply:
Sorry,  I should have been clearer. You should be able to build and configure libf3d example on arm I think.